storyblok/storyblok-design-system
# Fix int 1077 date picker last number

Added css border bottom to datepicker's hour and minutes selector, when multiple of 4.
## Pull request type

Jira Link: [INT-1077](https://storyblok.atlassian.net/browse/INT-1077)




## How to test this PR


1 - Click on schedule Publish
2 - Select a day, then go to time selection.
3 - The last minute should have a border botton.

## What is the new behavior?



- 
- 
- 

## Other information


[INT-1077]: https://storyblok.atlassian.net/browse/INT-1077?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
